package.json

```diff
@@ -7,7 +7,8 @@
     "node": "16.4.2"
   },
   "scripts": {
-    "start": "concurrently \"nodemon server/index.js\" \"cd client && npm start\"",
+    "client": "cd client && npm start",
```

Comment:
I figured out that we don't actually need concurrently, since we can just use these separate start scripts to run both the server and client at the same time by configuring our `launch.json` files a certain way. I'll share the setup I discovered in slack